src/index.css

```diff
@@ -0,0 +1,66 @@
+html {
+  box-sizing: border-box;
+  min-height: 100%;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+  padding: 0;
+  margin: 0;
+  top: 0;
+}
+
+body {
+  font-family: cursive;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* background: #371300; */
```

Comment:
- [ ] Please make sure there is no commented code. This makes hard to read the code.